apps/essence/server/services/db/repository/channel.ts

```
@@ -0,0 +1,35 @@
+      .update(channels)
+      .set(data)
+      .where(eq(channels.id, id))
+      .returning()
+
+    return channel
+  }
+
+  static async delete(id: string) {
+    await useDatabase().delete(channels).where(eq(channels.id, id))
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)